File: GARI_Umbria_2026-08-03/GiasNG/src/app/Model/attivita/risorse/RisorsaPersona.ts

import { RisorseUmane } from '../../anagrafiche/RisorseUmane';
import { RisorsaTimeSheet } from './RisorsaTimeSheet';

export class RisorsaPersona extends RisorsaTimeSheet {
    risorsaUmana: RisorseUmane;

    constructor() {
        super();
        this.classType = 'RisorsaPersona';
    }
}

File: GARI_Umbria_2026-08-03/DotNet/AgronicaCoreStd/AgronicaCoreModelsSTD/attivita/risorse/RisorsaPersona.cs
using AgronicaCoreModelsSTD.anagrafiche;
using System;
using System.Collections.Generic;
using System.Text;

namespace AgronicaCoreModelsSTD.attivita.risorse
{
    public class RisorsaPersona: RisorsaTimeSheet
    {
        public RisorseUmane risorsaUmana { get; set; }

        public RisorsaPersona()
        {
            classType = costanti.ClassType.RisorsaPersona;
        }
    }
}
